redbot/core/modlog.py

```diff
@@ -149,7 +150,9 @@ def __init__(
         """
         casetype = await get_casetype(self.action_type)
         title = "{}".format(
-            _("Case #{} | {} {}").format(self.case_number, casetype.case_str, casetype.image)
+            _("Case #{} | {} {}").format(
+                humanize_number(self.case_number), casetype.case_str, casetype.image
```

Comment:
I think in this circumstance we shouldn't use `humanize_number`. We use case numbers as input to other commands, this makes it slightly more annoying for users to copy+paste the case number